data/distro.yml

```diff
@@ -1,790 +1,612 @@
-      </li>
-      <li>
-        <h2>Add the Flathub repository</h2>
-        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
-        <terminal-command>
-          flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
-        </terminal-command>
-      </li>
-      <li>
-        <h2>Restart</h2>
-        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
-      </li>
-    </ol>
-
-- name: "Fedora"
-  logo: "fedora.svg"
-  info: >
-    <ol class="distrotut">
-      <!-- Flatpak has been installed by default on Fedora Workstation since F25. Previous versions are past end of life, so don’t need to be mentioned. -->
-      <p>Flatpak is installed by default on Fedora Workstation, Fedora Silverblue, and Fedora Kinoite. To get started, all you need to do is enable Flathub, which is the best way to get Flatpak apps. Just download and install the <a class="btn btn-default" href="https://dl.flathub.org/repo/flathub.flatpakrepo">Flathub repository file</a>.</p>
-      <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
-      <p>The above links should work on the default GNOME and KDE Fedora installations, but if they fail for some reason you can manually add the Flathub remote by running:</p>
+  steps:
+    - name: Install Flatpak
+      text: "
+      <p>To install Flatpak on Ubuntu 18.10 (Cosmic Cuttlefish) or later, simply run:</p>
+      <terminal-command>sudo apt install flatpak</terminal-command>
+      <p>With older Ubuntu versions, the official Flatpak PPA is the recommended way to install Flatpak. To install it, run the following in a terminal:</p>
+      <terminal-command>
+        sudo add-apt-repository ppa:flatpak/stable\n
+        sudo apt update\n
+        sudo apt install flatpak\n
+      </terminal-command>"
+    - name: Install the Software Flatpak plugin
+      text: "
+      <p>The Flatpak plugin for the Software app makes it possible to install apps without needing the command line. To install, run:</p>
+      <terminal-command>sudo apt install gnome-software-plugin-flatpak</terminal-command>
+      <p>Note: the Software app is distributed as a Snap since Ubuntu 20.04 and does not support graphical installation of Flatpak apps. Installing the Flatpak plugin will also install a deb version of Software and result in two Software apps being installed at the same time.</p>"
+    - name: Add the Flathub repository
+      text: "
+      <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
       <terminal-command>
         flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
-      </terminal-command>
-    </ol>
+      </terminal-command>"
+    - name: 'Restart'
```

Comment:
What about this (and other `name: 'Restart'` lines)?

Reply:
Missed, as I regexed it :D